src/Controllers/news.controller.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    Express,
+    Request,
+    Response,
+    NextFunction,
+    RequestHandler,
+} from 'express';
+import axios from 'axios';
+import cheerio from 'cheerio';
+
+export const getNewsMFI: RequestHandler = async (
+    req: Request,
+    res: Response,
+    next: NextFunction
+) => {
+    const HTMLData = await axios
+        .get('https://mfi.ug.edu.pl/wydzial/aktualnosci')
+        .then((res) => res.data)
+        .catch((err) => {
+            res.send(err);
+        });
+    const newsArray: {}[] = [];
+    const $ = cheerio.load(HTMLData);
+    const selectedElem: string = '.item-list > ul > li';
+    $(selectedElem).each((parentIndex, parentElem) => {
```

Comment:
maybe theres a way to use reduce method instead of loop?

Reply:
idk i searched the internet and it seems that everyone uses each or map with cheerio